src/routes/_header.svelte

```diff
@@ -16,14 +16,24 @@
 <script lang="ts">
   import { namespace } from '$lib/stores/namespace';
   import DataConvertorStatus from '$lib/components/data-convertor-status.svelte';
+  import NavigationLink from './_navigation-link.svelte';
   export let user: { name?: string; email?: string; picture?: string } = {};
 </script>
 
 <div
   id="header"
-  class="h-16 w-full px-6 flex flex-row items-center justify-between border-b-2"
+  class="h-16 w-full px-6 flex flex-row items-center justify-between bg-black border-b-2"
```

Comment:
Looking at the [mock ups](https://www.figma.com/file/u0hxNQYdSWagO8eOlXkgea/Temporal-Web-UI-v2?node-id=356%3A6371), it looks we're using `bg-gray-900` and not black.